src/data-sources/connections/application-insights/QueryTester.tsx

```diff
@@ -27,9 +28,9 @@ export default class QueryTester extends React.Component<IQueryTesterProps, IQue
   state: IQueryTesterState = {
     showDialog: false,
     query: '',
-    response: '',
+    response: {},
     loadingData: false,
-    responseHeight: 100
+    expandResponse: true
```

Comment:
This is the same name as the method name `this.expandResponse`, please update this value to responseExpanded or something similar